modules/stream.js

```diff
@@ -84,6 +84,26 @@ export class StreamViewStream extends StreamView {
 		}
 	}
 
+	/**
+	 * @override
+	 */
+	async setChatMode(mode) {
+		super.setChatMode(mode);
+
+		if (mode == StreamViewOptions.ChatVisibility.ALWAYS) {
```

Comment:
Prefer `switch`  to `if`/`else if` chain.

Reply:
yeah sorry, that was just sloppy of me, as long as its more than a single if/else, I prefer switch too.